ui/admin/app/components/form/credential/json/index.hbs

```diff
@@ -11,22 +11,27 @@
   as |form|
 >
 
-  <form.input
-    @name='name'
+  <Hds::Form::TextInput::Field
     @value={{@model.name}}
-    @label={{t 'form.name.label'}}
-    @error={{@model.errors.name}}
-    @helperText={{t 'form.name.help'}}
-    as |field|
+    @isInvalid={{@model.errors.name}}
+    @isOptional={{true}}
+    name='name'
```

Comment:
non-blocking small change: i think `name` attr should be move to the top like the other fields

Reply:
@lisbet-alvarez yes thank you, incoming in next commit